base/commands/shell.go

```diff
@@ -212,6 +218,17 @@ func convertStatement(stmt string) (string, error) {
 	return stmt, nil
 }
 
+func maybePrintNewerVersion(ec plug.ExecContext) {
```

Comment:
I cannot add it as a boolean variable:
```
 -X definition
        add string value definition of the form importpath.name=value
```

Reply:
That's right, `internal. EnableCheckVersion` should be a string, so when its value is `false`, version check should be skipped.

Reply:
In the rest of the comments we decided to use `internal.SkipUpdateCheck` instead of `internal.EnableCheckVersion` and use `0` and `1` for consistency with the environment variable.